common/v2/components/AssetIcon.tsx

```diff
@@ -1,19 +1,22 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import styled from 'styled-components';
-import manifest from 'cryptocurrency-icons/manifest.json';
+import { RatesContext } from 'v2/services';
+import { TUuid } from 'v2/types';
+import genericIcon from 'assets/generic.svg';
 
-import { IAsset, TSymbol } from 'v2/types';
 // Relies on https://github.com/atomiclabs/cryptocurrency-icons using fixed version number through CDN
 // @TODO: We should be using our own sprite served over a trusted CDN
-const baseURL = 'https://cdn.mycryptoapi.com/v1/icons';
+const baseURL = 'https://mycryptoapi.com/api/v1/images';
```

Comment:
can we use a constant for 'mycryptoapi.com/api/v1'? It would make it easier to update the versions, handle CORS declaration and adapt to changes in the domain name.

Reply:
Constant was already added: https://github.com/MyCryptoHQ/MyCrypto/blob/master/common/v2/config/constants.ts#L15